administrator/components/com_admin/script.php

```diff
@@ -545,4 +546,189 @@ private function cleanJoomlaCache()
 		$model->setState('client_id', 1);
 		$model->clean();
 	}
+
+	private function installWorkflow()
+	{
+		if (!JComponentHelper::isEnabled('com_workflow'))
+		{
+			return;
+		}
```

Comment:
What happens if workflow already exists but is disabled? or is it a core extension which cannot be disabled?

Reply:
Migration is from where there are no-workflow status to workflow state. So at this point com_workflow has to be enabled